docs/usage/self-hosting.md

```diff
@@ -29,16 +30,17 @@ $ docker run --rm renovate/renovate:24.10
 $ docker run --rm renovate/renovate:24
 ```
 
-(Please look up what the latest actual tags are though, do not use the above literally).
+Do not use the example tags listed above, as they will be out-of-date.
+Go to [renovate/renovate tags](https://hub.docker.com/r/renovate/renovate/tags) to grab the latest tagged release from Renovate.
```

Comment:
```suggestion
```

We now have a bot that updates the `renovate/renovate` Docker tags.

Do we still need the warning and the link to the `renovate/renovate` DockerHub?

Reply:
sure, as they wil only be updated weekly.